### ISSUE TITLE
Connexion : le bouton radio n'est pas centré quand il est coché

### ISSUE THREAD

**Issue by juliehirtz** [TruAcademy/TruAcademy#3]
![image](https://cloud.githubusercontent.com/assets/14889153/12760473/d118d756-c9e6-11e5-8791-bf936dbc5bec.png)

**Comment by gealex** [comment]
Yes, ca fait ça bizarrement de temps en temps. Je suis entrain de voir comment je peux améliorer.
